mmv1/third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -859,6 +859,31 @@ func TestAccBigQueryExternalDataTable_parquet(t *testing.T) {
 	})
 }
 
+func TestAccBigQueryExternalDataTable_objectTable(t *testing.T) {
+	t.Parallel()
+
+	bucketName := testBucketName(t)
+	objectName := fmt.Sprintf("tf_test_%s.csv", randString(t, 10))
+
+	datasetID := fmt.Sprintf("tf_test_%s", randString(t, 10))
+	tableID := fmt.Sprintf("tf_test_%s", randString(t, 10))
+	connectionID := fmt.Sprintf("tf_test_%s", randString(t, 10))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckBigQueryTableDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccBigQueryTableFromGCSObjectTable(connectionID, datasetID, tableID, bucketName, objectName),
+			},
+			{
+				Config: testAccBigQueryTableFromGCSObjectTableMetadata(connectionID, datasetID, tableID, bucketName, objectName),
```

Comment:
Can you add a third step that goes back to the original `testAccBigQueryTableFromGCSObjectTable` config? That will test that unsetting the metadata_cache_mode works

Reply:
added